src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -539,7 +531,7 @@ public Object onTraversalSuccess() {
         } catch (final FileNotFoundException fe) {
             throw new UserException("Unable to write callset map JSON file " + callsetMapJSONFile.getAbsolutePath(), fe);
         }
-        try {
+	try {
```

Comment:
Fix indentation here (there may be literal tabs here that need converting to spaces).

Reply:
done